index.rst

```diff
@@ -366,75 +214,46 @@ an alphabetic prefix (eg., ``v``).  This has the effect of changing the *eups*
 version strings as ``lsst-build`` sets the *eups* product version based on the
 most recent git ref that has an *integer* as the first character.
 
-As consequene of this behavior is that the final git tag **must** be present
+As consequence of this behavior is that the final git tag **must** be present
 prior to the production of ``eupspkg``/*eups tag*.
 
-Final tag
-^^^^^^^^^
-
-XXX failures are now fatal...
-
-Now it's time to lay down the final git tag. For repositories that have already
-been branched with the ``14.0`` ref, that will fail, which is fine.
-
-This is mostly a repeat of the process for laying down the candidate tag but
-this time we use numeric tags so that eups will see them:
-
-.. code-block:: bash
-
-   # tag repos involved in the final candidate as the final build
-
-   github-tag-version \
-     --org lsst \
-     --allow-team 'Data Management' \
-     --allow-team 'DM Externals' \
-     --external-team 'DM Externals' \
-     --deny-team 'DM Auxilliaries' \
-     --debug \
-     --candidate 'v14_0_rc2 \
-     '14.0' 'b3176'
-
-
-Release build
-^^^^^^^^^^^^^
+Build and Publish
+^^^^^^^^^^^^^^^^^
 
-- Submit the run-rebuild job with your parameters (eg. ``14.0`` ``v14.0``)
+Run the jenkins `release/official-release <official-release>`_ job.
 
-- At this point you should not be seeing master-g type references as eups
-  versions. Everything should have a tag-derived version such as ``14.0`` if
-  they are a DM repo and their semantic tag (eg. ``pyfits 3.0``) if they are
-  external.  If you see one, you need to chase down why. The only situation
-  that should happen is if a third party but a branch is used for LSST
-  development that lacks any other type of semantic versioning (in the ``14.0``
-  release this included starlink_ast and jointcal_cholmod.
+For input **must only be** the latest ``rc`` tag.
+Example 1:
 
-- Note your final ``bNNNN`` number for the publish (either from the build log
-  or by looking at the next of the annotated ``14.0`` tag on any repo eg. afw).
+.. code-block:: text
 
-- Submit the run-publish job making sure you have selected ``package`` and not
-  ``git`` as the option.
+   SOURCE_GIT_REFS: v42.0.0.rc6
+   RELEASE_GIT_TAG: 42.0.0
+   O_LATEST: false
 
-Branching lsst
-^^^^^^^^^^^^^^^
+Branch ``newinstall.sh`` repo
```

Comment:
newinstall.sh shall be branched when the first release candidate is created, and added a final official tag when the release is concluded.

Reply:
It should be branched after because, at present, the jenkins job's always use `newinstall.sh` from `master`.

Reply:
I'm going to mark this as resolved as the newinstall docker image build and tarball pipeline would need to be modified to look for `newinstall.sh` on a branch for it to be safe to branch in advance.